.github/workflows/build.yml

```diff
@@ -215,18 +200,10 @@ jobs:
         run: |
           ./autogen.sh --enable-option-checking=fatal --enable-debug-messages ${{ matrix.gcrypt }} ${{ matrix.msan }} ${{ matrix.pcre }} ${{ matrix.maxminddb }} --enable-tls-sigs ${{ matrix.global_context}}
       - name: Build nDPI
-        if: ${{ !startsWith(matrix.os, 'macos-14') && !startsWith(matrix.os, 'macos-15') }}
         run: |
           make -j all
           make -C example ndpiSimpleIntegration
           make -C rrdtool
-      - name: Build nDPI (MacOS M1)
-        if: startsWith(matrix.os, 'macos-14') || startsWith(matrix.os, 'macos-15')
-        run: |
-          make -j all
-          make -C example ndpiSimpleIntegration
```

Comment:
You do not want to build it at least once on Mac?

Reply:
macos builds use the standard step "Build nDPI" just above, as well as ubuntu builds; no more need for a specific step....
We will build on macos-13,14,15 (standard compilation) and on latest with some external dependencies